Content.Client/Cargo/UI/BountyEntry.xaml.cs

```diff
@@ -38,6 +39,20 @@ public BountyEntry(CargoBountyData bounty)
         DescriptionLabel.SetMarkup(Loc.GetString("bounty-console-description-label", ("description", Loc.GetString(bountyPrototype.Description))));
         IdLabel.SetMarkup(Loc.GetString("bounty-console-id-label", ("id", bounty.Id)));
 
-        PrintButton.OnPressed += _ => OnButtonPressed?.Invoke();
+        PrintButton.OnPressed += _ => OnLabelButtonPressed?.Invoke();
+        SkipButton.OnPressed += _ => OnSkipButtonPressed?.Invoke();
+    }
+
+    public void UpdateSkipButton(TimeSpan untilNextSkip)
```

Comment:
you can make this update in real time if you
- store `untilNextSkip` inside of this class
- just call this function via `FrameUpdate()`

that way you can watch the timer tick down.